networkit/cpp/numerics/LAMG/MultiLevelSetup.cpp

```diff
@@ -1,247 +0,0 @@
-/*
```

Comment:
@angriman has more knowledge of the LAMG than I do. What do we lose if we remove these specializations? (vs. the generic implementation in the header?)